### PR TITLE
Unify frontend and backend development servers

### PR DESCRIPTION
**Pull request by jacebrowning**
https://www.vistimo.com/feature/9831ec2a-f3ed-4188-ac1d-6a577c1c010c

-----

`$ make run` should allow you to access:

- [ ] http://localhost:5000/
- [ ] http://localhost:5000/home
- [ ] http://localhost:5000/api
- [ ] http://localhost:5000/admin

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,20 +61,27 @@ endif
...
-	pipenv shell -c "bin/post_compile; exit \$$?"
-	pipenv shell -c "heroku local release; exit \$$?"
-	pipenv shell -c "heroku local web; exit \$$?"
+# TODO: Emulate the production server
+# .PHONY: run-prod
+# run-prod: install ## Run the application (emulate production)
+# 	pipenv shell -c "bin/pre_compile; exit \$$?"
+# 	pipenv shell -c "bin/post_compile; exit \$$?"
+# 	pipenv shell -c "heroku local release; exit \$$?"
+# 	pipenv shell -c "heroku local web; exit \$$?"
```
Comment (jacebrowning) [inline Makefile]: I'll come back to this when the frontend is getting built on Heroku.

**Comment by diiq** [comment]
The frontend half has a README that explains what needs to be installed as a prerequisite, how to run the package manager, and how to run the local server, among other things. 

Similar info should be written up about the API; and all of it should probably move to the top level README.

**Comment by diiq** [comment]
Following the readme, I got to "make run" and was told the database doesn't exist.

Are there scripts for creating/seeding/dropping the database and test database? Or is it easiest to do that manually in djangoworld?

**Comment by jacebrowning** [comment]
@diiq I think we should just create it manually for now. I added instructions to the README.

**Comment by diiq** [comment]
It's a bummer to lose color output from the typescript compiler (normally type errors and compile errors are red, etc) but in the interest of progress, let's merge it.

**Pull request merged**